packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -88,11 +88,16 @@ class CupertinoSwitch extends StatefulWidget {
   _CupertinoSwitchState createState() => new _CupertinoSwitchState();
 
   @override
-  void debugFillDescription(List<String> description) {
-    super.debugFillDescription(description);
-    description.add('value: ${value ? "on" : "off"}');
-    if (onChanged == null)
-      description.add('disabled');
+  void debugFillProperties(List<DiagnosticsNode> description) {
+    super.debugFillProperties(description);
+    description.add(new FlagProperty(
+      'value',
+      value: value,
+      ifTrue: 'on',
+      ifFalse: 'off',
+      showName: true,
+    ));
+    description.add(new ObjectFlagProperty<ValueChanged<bool>>('onChanged', onChanged, ifNull: 'disabled'));
```

Comment:
I would be more aggressive about putting these on one line. This one fits fine as far as I can tell, for example; in fact it's one character less than the one you do have on one line here:
```dart
    description.add(new FlagProperty('value', value: value, ifTrue: 'on', ifFalse: 'off', showName: true));
    description.add(new ObjectFlagProperty<ValueChanged<bool>>('onChanged', onChanged, ifNull: 'disabled'));
```